src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx

```diff
@@ -160,4 +187,4 @@ MissionDetailsCard.propTypes = {
   mission: PropTypes.object.isRequired,
 };
 
-export default MissionDetailsCard;
+export default withRouter(withFirestore(MissionDetailsCard));
```

Comment:
Doesn't look like you're using the router or firestore as a `prop` for this component so can get rid of the HOCs